Week1/homework/js-exercises/9_logRemainder.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+let x = 7;
+x = x % 3;
+console.log("Final value of x:",x);
+let y = 21;
+y = y % 4;
+console.log("Final value of y:",y);
+let z = 13;
+z = z % 2;
+console.log("Final value of z:",z); 
```

Comment:
You are asked to

> For each of these, write in comments what the answer is followed by how you came to that conclusion

For example: `console.log("Final value of x:",x); // Answer is 1 because it is the remainder of dividing 7 by 3`